src/map/src/combat/combat.cpp

```diff
@@ -418,6 +504,25 @@ void Combat::revive(EntitySystem& entitySystem, Entity entity, const RoseCommon:
   entitySystem.teleport_entity(entity, std::get<1>(dest), std::get<2>(dest), std::get<0>(dest));
 }
 
+int32_t Combat::get_exp_to_level(int level) {
```

Comment:
prob should be a int64_t here no?

Reply:
the level component is int32_t, only made that match that.

Reply:
Anyway I have fixed it in the latest

Reply:
I had put in32_t here because the packets expect that 🤷‍♂